core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/dot-experiments-configuration.component.html

```diff
@@ -4,22 +4,17 @@
         [title]="vm.experiment?.name"
         (goBack)="goToExperimentList()"
         backUrl="{{ '/edit-page/experiments/' + vm.pageId }}"
-    ></dot-experiments-header>
+    >
+    </dot-experiments-header>
 
     <div class="dot-experiment-config-content">
         <ng-container *ngIf="!vm.isLoading; else loadingTpl">
-            <dot-experiments-configuration-experiment-status-bar
-                [status]="vm.experiment.status"
-            ></dot-experiments-configuration-experiment-status-bar>
+            <dot-experiments-configuration-experiment-status-bar [status]="vm.experiment.status">
+            </dot-experiments-configuration-experiment-status-bar>
 
-            <p>
-                {{ vm.experiment.description }}
-            </p>
+            <p>{{ vm.experiment.description }}</p>
 
-            <dot-experiments-configuration-variants
-                [trafficProportion]="vm.experiment.trafficProportion"
-            >
-            </dot-experiments-configuration-variants>
+            <dot-experiments-configuration-variants></dot-experiments-configuration-variants>
```

Comment:
Pass here from the store and receive here to update the store, don't use the store around.